dashboard/config/scripts/levels/Multi Strings Images 1.level

```diff
@@ -27,7 +27,7 @@
       }
     ],
     "title": "Strings and Images Multiple Choice",
-    "content1": "http://staging.studio.code.org/script_assets/msm/green-rect.png",
+    "content1": "http://studio.code.org/script_assets/msm/green-rect.png",
```

Comment:
Are we sure that these assets are available in the production environment?  I don't know how that pipeline works.